data/fields/seamark/beacon_isolated_danger/shape.json

```diff
@@ -1,5 +1,41 @@
 {
     "key": "seamark:beacon_isolated_danger:shape",
     "type": "combo",
-    "label": "Shape"
+    "label": "Shape",
+    "strings": {
+        "options": {
+            "pile": {
+                "title": "Pile Beacon",
+                "description": "A long, heavy pile or section of metal, wood or concrete, driven into the seabed, used as a navigational aid."
```

Comment:
Are these meanings documented anywhere?

I checked https://wiki.openstreetmap.org/wiki/Key:seamark:beacon_isolated_danger:shape and see only few of them therre